internal/resources/policy/kind/quota/constants.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package policykindquota
+
+import (
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy"
+	reciperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/quota/recipe"
+)
+
+const (
+	ResourceName     = "tanzu-mission-control_quota_policy"
+	typeDefaultValue = "namespace-quota-policy"
```

Comment:
Should this also be renamed into just quota_policy

Reply:
We cannot do that as we get `400 Bad Request, response: {"error":"InvalidArgument: permission denied for policy type 'quota-policy'","code":3,"message":"InvalidArgument: permission denied for policy type 'quota-policy'` for this as in the API request, it accepts- `namespace-quota-policy`. Reference: screenshot in above comment. Only resource name can be changed, `typeDefaultValue` is mapped to  the Type in the API - `VmwareTanzuManageV1alpha1CommonPolicySpec` where Type is `Type of the policy object`.

Reply:
can you rename the variable so that the variable says this is being used in the API as policy type

Reply:
I have replaced `typeDefaultValue` to `typePolicy` for all policies and added a comment as well.